### DISCUSSION TITLE
Eclipse import dbeaver22.2.0 occur some maven problems

### DISCUSSION THREAD

**Discussion by ccczhouxin** [dbeaver/dbeaver#17687] [DBeaver Development]
### Description

![image](https://user-images.githubusercontent.com/43168824/191207189-30bdf08f-4a6b-460c-868a-46b5f99ecbe3.png)
Plugin execution not covered by lifecycle configuration: org.apache.maven.pluginsmaven-clean-plugin2.5xclean (execton: default-clean-1, phase intialize)
I fllow https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse ,in the last occur the problem

### DBeaver Version

Community Edition 22.2.0

### Operating System

windows 64

**Comment by notpdpton** [comment]
I have the same issue, and I tried in VM with clear env. But it still not work. ERROR as follow:

![image](https://user-images.githubusercontent.com/20450731/191217905-eb8c0e67-5995-414f-a14f-c66a5d24bb1b.png)

## Environment

- Windows 10 21H2 x64 (CPU 6 core, Memory 8 GiB)
- Adoptuim OpenJDK 11.0.16
- Maven 3.6.3
- Eclipse RCP 2021-12
- DBeaver Branch release_22_2_0

All the step follows: [Develop in Eclipse](https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse)

## More

The default JRE is complete JDK. I just rename it to `jre` and put it into eclipse root directory.

![image](https://user-images.githubusercontent.com/20450731/191220642-6e12a3bf-31cd-4865-b100-c231c0054aa1.png)

Maven had changed to `3.6.3`

![image](https://user-images.githubusercontent.com/20450731/191221465-cca9474a-9d40-4577-84d7-95d71c728eb2.png)

I tried update project, but it not work too.

![image](https://user-images.githubusercontent.com/20450731/191222879-98e0da5c-463d-48d7-bcc9-e21158712bee.png)
